compress/compress.go

```diff
@@ -20,6 +20,9 @@ import (
 	"io"
 )
 
+var gzipMagicHeader = []byte{'\x1f', '\x8b'}
+
+// Compress compresses the input stream.
```

Comment:
I believe this is now unused (though I didn't delete it).  Can we move `IsCompressed` into `v1util` and delete this?